src/modules/atoms/link/link.mixin.pug

```diff
@@ -0,0 +1,4 @@
+
+mixin link(text, destination)
+
+  a(href=destination)= text
```

Comment:
* add a class to the tag (e.g. link)
* add `attributes(attributes)` to the link, if mixins are called with extra arguments